.gitignore

```diff
@@ -2,7 +2,6 @@
 .Rhistory
 .RData
 revdep/.cache.rds
-.Rbuildignore
```

Comment:
Not sure why this was here but `.Rbuildignore` is being tracked, so it has had no effect anyway.